src/pages/webhooks/release-notes.md

```diff
@@ -9,6 +9,20 @@ keywords:
 
 These release notes describe the latest version of Adobe Commerce Webhooks.
 
+## Version 1.2.0
+
+### Release date
+
+February 7, 2024
+
+### Enhancements
+
+* Added a new module `AdobeCommerceWebhooksAdminUi` for managing webhooks from the Admin UI
+
+* Added an ability to modify and create webhooks from the Admin UI
```

Comment:
Wondering if we should mention xml-defined webhooks specifically just to make it clearer that the admin UI can be used to extend/override xml config

Reply:
Thanks, updated it. But don't know how to rephrase it in a better way. @keharper please take a look